core_v2/sources/domain_e2e_tests.move

```diff
@@ -467,4 +468,96 @@ module aptos_names_v2::domain_e2e_tests {
             assert!(is_owner, 1);
         };
     }
+
+    #[test(
+        aptos_names = @aptos_names,
+        aptos_names_v2 = @aptos_names_v2,
+        user = @0x077,
+        aptos = @0x1,
+        rando = @0x266f,
+        foundation = @0xf01d
+    )]
+    fun test_migration(
```

Comment:
very minor thing. I noticed that the old tests are all names `**_test()` and I do like the convention with `test_*` more. should we unify everything here?